app/src/components/explorer/SearchHeader.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div class="explorer_page_header">
+        <md-card style="padding:2rem; margin:2rem;">
+        <div class="search_box_form">
+            <div class="form__group search_box_form-item-1">
+                <input type="text" ref="search_input" class="form__input form__input--adjust" placeholder="Search" name="search" id="search" required v-model="searchWord" />
+                <label htmlFor="search" class="form__label search_box_form_label">Search</label>
+            </div>
+            <div class="form__group search_box_form-item-2">
+                <button type="submit" class="btn btn--primary btn--noradius search_box_form_btn">Search</button>
+            </div>
+        </div>
+        <div style="text-align:right"> Query "{{searchWord}}" found 12345 results </div>
+        </md-card>
+        <md-tabs class="btn--primary">
+            <md-tab :md-label="'Articles (' + articles + ')'" @click="setResultsTab('Articles')"></md-tab>
+            <md-tab :md-label="'Samples (' + samples + ')'" @click="setResultsTab('Samples')"></md-tab>
+            <md-tab :md-label="'Images (' + images + ')'" @click="setResultsTab('Images')"></md-tab>
+            <md-tab :md-label="'Charts (' + charts + ')'" @click="setResultsTab('Charts')"></md-tab>
+            <md-tab :md-label="'Materials (' + materials + ')'" @click="setResultsTab('Materials')"></md-tab>
+            <!-- <md-tab :md-label="'Other (' + other + ')'" @click="setResultsTab('Other')"></md-tab> -->
```

Comment:
I see what you doing here. Good job! It's a great idea just in case we want to enable that